server/middleware/get-syndication-licence-for-user.js

```diff
@@ -54,6 +56,11 @@ module.exports = exports = async (req, res, next) => {
 
 		res.locals.licence = syndicationLicence;
 
+		if(!res.locals.hasGraphicSyndication) {
+			res.locals.hasGraphicSyndication = hasGraphicsAccess(syndicationLicence.products);
```

Comment:
```suggestion
            const {GRAPHIC_SYNDICATION_PRODUCT_CODE} = require('config');
			res.locals.hasGraphicSyndication = syndicationLicence.products.some(product => product.code === GRAPHIC_SYNDICATION_PRODUCT_CODE);
```

This logic can be contained in the same file

Reply:
True did this becuase I thought I was going to need to repeat. It's tested as a standalone helper so don't think theres any harm in leaving it so.